src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
-}
+  const robot = {
+    name,
+    wheels,
+    version,
+    x: 0,
+    y: 0,
+    get info() {
+      return (`name: ${
+        this.name
+      }, chip version: ${
+        this.version
+      }, wheels: ${
+        this.wheels
+      }`);
+    },
+    get location() {
+      return `${this.name}: x=${this.x}, y=${this.y}`;
+    },
+    goForward(step = 1) {
+      if (step > 0) {
+        this.y += step;
+      }
+
+      return this;
+    },
+    goBack(step = 1) {
+      if (step > 0) {
+        this.y -= step;
+      }
+
+      return this;
+    },
+    goRight(step = 1) {
+      if (step > 0) {
+        this.x += step;
+      }
+
+      return this;
+    },
+    goLeft(step = 1) {
+      if (step > 0) {
+        this.x -= step;
+      }
+
+      return this;
+    },
+    evacuate() {
+      this.x = 1400;
+      this.y = 500;
+    },
+    get coords() {
```

Comment:
```suggestion
    },
    
    get coords() {
```
Добавляй пробел между методами для читабельности